api/src/test/resources/answer-challenge-identify-first-response.json

```diff
@@ -0,0 +1,229 @@
+{
+  "version": "1.0.0",
+  "stateHandle": "02_SASNV3rEhVtB1v3sW5dkSvDCproFFJLi2p6ecer",
+  "expiresAt": "2021-05-17T15:20:45.000Z",
+  "intent": "LOGIN",
+  "remediation": {
+    "type": "array",
+    "value": [
+      {
+        "rel": ["create-form"],
+        "name": "select-authenticator-enroll",
+        "href": "https://dev-90108051.okta.com/idp/idx/credential/enroll",
```

Comment:
Should we make these knowingly fake, so that we don't accidentally call out to a real server, or confuse readers?

Reply:
my mistake. fixed.

Reply:
Good catch.. also please double check none of your UTs have real URLs in the mock responses.